_couches/abhi.md

```diff
@@ -3,11 +3,12 @@ name: Abhijeet Goel
 city: Rishikesh
 country: IN
 region: Najibabad
-email: ninjamonk@protonmail.com
+email: ninjamonk@pm.me
 github: aNinjaMonk
 twitter: aninjamonk
+website: www.ninjamonk.in
```

Comment:
Can you please prefix this with  `http://` or `https` please?

Build failing for invalid link: https://travis-ci.org/hackercouch/hackercouch/builds/580996829

Reply:
Updated.